phoenix-core/src/main/java/org/apache/phoenix/compile/ScanRanges.java

```diff
@@ -694,4 +694,138 @@ public TimeRange getRowTimestampRange() {
         return rowTimestampRange;
     }
 
+    public List<KeyRange> getRowKeyRanges(){
```

Comment:
Could you add comments (function spec?) for this function?

Reply:
I have added function level comments. Some basic testing was already included in the unit tests.  As this code snippet does nothing, I can not include functional tests.